test/errors/redef-detailed.good

```diff
@@ -0,0 +1,15 @@
+─── error in redef.chpl:1 [Redefinition] ───
+  'x' has multiple definitions.
+      |
+    1 | var x =
```

Comment:
I would personally like to see this code snippet called out as "First definition:" or similar, in at least detailed mode but I would argue also in brief mode.